src/patternfly/components/CalendarMonth/calendar-month.scss

```diff
@@ -136,21 +143,34 @@
   padding-inline-end: var(--#{$calendar-month}__dates-cell--PaddingInlineEnd);
   text-align: center;
 
-  &::before {
+  &::before,
+  &::after {
     position: absolute;
     inset-block-start: var(--#{$calendar-month}__dates-cell--before--InsetBlockStart);
     inset-block-end: var(--#{$calendar-month}__dates-cell--before--InsetBlockEnd);
     inset-inline-start: var(--#{$calendar-month}__dates-cell--before--InsetInlineStart);
     inset-inline-end: var(--#{$calendar-month}__dates-cell--before--InsetInlineEnd);
     content: "";
+  }
+
+  &::before {
     background-color: var(--#{$calendar-month}__dates-cell--before--BackgroundColor);
   }
 
+  &::after {
+    border: solid var(--#{$calendar-month}__dates-cell--after--BorderColor);
+    border-width: var(--#{$calendar-month}__dates-cell--after--BorderBlockStartWidth) 0 var(--#{$calendar-month}__dates-cell--after--BorderBlockEndWidth);
+  }
```

Comment:
Does this bother anyone else enough to break convention and move it to the `::before`? 

<img width="639" height="367" alt="Image" src="https://github.com/user-attachments/assets/3f260ea6-887c-4b03-b209-4c5a39d11804" />

Reply:
Hmm yeah the disabled ones cause a weird visual effect when the border is beneath. Given that, I'd stick with how you have it as the lesser of evils 😈